cmd/preflight/cmd/root.go

```diff
@@ -48,7 +51,13 @@ func rootCmd() *cobra.Command {
 }
 
 func Execute() error {
-	return rootCmd().ExecuteContext(context.Background())
+	zapLog, err := zap.NewDevelopment()
+	if err != nil {
+		panic(fmt.Sprintf("who watches the watchmen (%v)?", err))
```

Comment:
I imagine this is here for WIP purposes, but just wanted to point it out to be sure.

Reply:
Yes. Copy pasta.